src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-agent-gradle.mdx

```diff
@@ -0,0 +1,220 @@
+
+1. Add the following code to your `build.gradle` files found in the Gradle & Android Studio tab. You need to update the top-level build snippet and each sub-module that you want to instrument. 
+
+  <CollapserGroup>
+    <Collapser
+      id="project-level"
+      title="Project level `build.gradle` file:"
+    >
+      In this example, `<AGENT_VERSION>` represents your agent version number. See the [agent release notes](/docs/release-notes/mobile-release-notes/android-release-notes), and use the latest version.
+
+      ```groovy
+      buildscript {
+        repositories {
+          mavenCentral()
+        }
+
+        dependencies {
+          classpath "com.newrelic.agent.android:agent-gradle-plugin:<AGENT_VERSION>"
+        }
+      }
+      ```
+    </Collapser>
+
+    <Collapser
+      id="app-level"
+      title="App level `build.gradle` file:"
+    >
+      In this example, `<AGENT_VERSION>` represents your agent version number. See the [agent release notes](/docs/release-notes/mobile-release-notes/android-release-notes), and use the latest version.
+
+      ```groovy
+      repositories {
+        mavenCentral()
+      }
+
+      apply plugin: 'android'
+      apply plugin: 'newrelic'
+
+      dependencies {
+        implementation 'com.newrelic.agent.android:android-agent:<AGENT_VERSION>'
+      }
+      ```
+    </Collapser>
+  </CollapserGroup>
+
+2. Set app permissions. Ensure that your Android app requests `INTERNET` and `ACCESS_NETWORK_STATE` permissions by adding these lines to your `AndroidManifest.xml` file:
+
+  ```xml
+  <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+  ```
+
+3. Start the Android agent. In your default Activity found in your manifest, import the `NewRelic` class:
+
+  ```
+  import com.newrelic.agent.android.NewRelic;
+  ```
+
+  We do not support starting the mobile monitoring agent in other classes, as that can cause unexpected or unstable behavior.
+
+4. After you've imported the `NewRelic` class, you need to add an additional snippet to the `onCreate()` method, which includes your app token, which is generated in the guided install. The snippet looks like this:
+
+  ```java
+  NewRelic.withApplicationToken("<GENERATED_TOKEN>").start(this.getApplicationContext());
+  ```
+
+5. If you're using minidification (for example, like with ProGuard or Dexguard), you need to add `newrelic.properties` file to your app-level directory (projectname/app). This step only applies to ProGuard and DexGuard users. 
+
+  ```
+  com.newrelic.application_token=<GENERATED_TOKEN>
+  ```
+
+  To finish set up for minification, follow the steps in [configure ProGuard or DexGuard for Android apps](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps).
```

Comment:
It's minification here and minidification here, I assume they're supposed to be the same term?